common/changes/@kadena/pactjs-cli/fix-pactjs-cli-readme_2023-06-27-10-05.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@kadena/pactjs-cli",
+      "comment": "Updated the README.md to have correct example usages",
+      "type": "none"
```

Comment:
FYI; Wasn't 100% sure whether we would like to have these README changes published as well, so for now chose `"none"`